src/Documentation/Tutorials.js

```diff
@@ -10,8 +10,8 @@ export default function Tutorials({ compact, tutorials }) {
 
   return (
     <Wrapper compact={compact}>
-      {tutorialsData.map(([key, value]) => {
-        const ButtonComponent = icons[key] || ExternalButton
+      {tutorialsData.map(([k, value]) => {
```

Comment:
why do we change names aggressively? human readable names (like key) are good enough when they stay short and to the point :)

Reply:
Oh, this one is because the JSX inside also has a `key` prop, which is assigned to `{value}` so I thought that was confusing and renamed the key argument or Array.map. I thought single-character variables for things like loops and array keys were a common (ok) practice but I can change it to something or revert this if you prefer, please lmk. Thanks

Reply:
I thought single-character variables for things like loops and array keys were a common (ok) practice but I can change it to something else or revert this if you prefer, please lmk. Thanks

Reply:
Sure! np. was just curious ... it's totally fine it the name overlaps and might be confusing 👍